pymech/readma2.py

```diff
@@ -0,0 +1,76 @@
+def readma2(fname,ldim):
```

Comment:
```suggestion
import struct

import numpy as np
from .log import logger


def readma2(fname,ldim):
```

You forgot the imports :D